plugins/scaffolder-backend/src/lib/templating/SecureTemplater.ts

```diff
@@ -102,13 +119,22 @@ export type SecureTemplateRenderer = (
 
 export class SecureTemplater {
   static async loadRenderer(options: SecureTemplaterOptions = {}) {
-    const { parseRepoUrl, cookiecutterCompat } = options;
-    let sandbox = undefined;
+    const { parseRepoUrl, cookiecutterCompat, additionalFilters } = options;
+    const sandbox: Record<string, any> = {};
```

Comment:
There is a consequence here that, if `parseRepoUrl` is `undefined` and no `additionalFilters` are passed in, we will pass `sandbox` into the VM as `{}` instead of the previous `undefined`. I assume this is okay, but if there was a particular reason why that is bad, I can make the changes necessary to prevent it.

Reply:
I think it's alright to pass in `{}`